coveralls.sh

```diff
@@ -7,16 +7,6 @@ DIR_SOURCE="$(find . -maxdepth 10 -type f -not -path '*/vendor*' -name '*.go' |
 BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
 echo "Using branch $BRANCH"
 
-pattern="refactor_*"
```

Comment:
I thought we only use this for cosi & cothority; was there a case where I missed that ( cc @nikkolasg )